### PR TITLE
Fix bugs in pointcloud about point tools

### PR DESCRIPTION
**Pull request by MMMMoriaty**
1.support zoom event when there is a remark layer
2.sync zoom levels in different tools when dragging
3.change the default size of sphere in 3d view

### CODE REVIEW

File: packages/lb-annotation/src/core/scheduler.ts [hunk 1]
```diff
@@ -346,5 +346,30 @@ export class ToolScheduler implements IToolSchedulerOperation {
...
+    this.container.removeEventListener('wheel', this.onWheel);
+  }
+
+  public onWheel(e: MouseEvent) {
+    e.preventDefault();
+    e.stopPropagation();
+    if (this.toolOperationList.length !== 0) {
+      const lastOneIndex = this.toolOperationDom.length - 1;
+      // the last one operation is the active one
+      this.toolOperationList[lastOneIndex].onWheel(e);
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/scheduler.ts:372]: Is there any problem in annotatiing ? It will trigger twice.（ zoom more quickly ?）

Comment (MMMMoriaty) [reply packages/lb-annotation/src/core/scheduler.ts:372]: If there is no remark layer, this event will be blocked by 'preventDefault' in 'onWheel' events in basicToolOperation, so it wont be triggered twice

**Pull request merged**